contracts/ShortNameClaims.sol

```diff
@@ -82,80 +106,193 @@ contract ShortNameClaims is Ownable {
+     * @dev Submits a claim for match on name+tld (eg, foo.tv -> footv).
      *      Claimants must provide an amount of ether equal to 365 days'
      *      registration cost; call `getClaimCost` to determine this amount.
      *      Claimants should supply a little extra in case of variation in price;
      *      any excess will be returned to the sender.
      * @param name The DNS-encoded name of the domain being used to support the
      *             claim.
      * @param claimant The address of the claimant.
+     * @param email An email address for correspondence regarding the claim.
      */
-    function submitCombinedClaim(bytes memory name, address claimant) public payable {
+    function submitCombinedClaim(bytes memory name, address claimant, string memory email) public payable {
         bytes memory firstLabel = bytes(getLabel(name, 0));
         bytes memory secondLabel = bytes(getLabel(name, 1));
         Buffer.buffer memory buf;
         buf.init(firstLabel.length + secondLabel.length);
         buf.append(firstLabel);
         buf.append(secondLabel);
 
-        handleClaim(string(buf.buf), name, claimant);
+        handleClaim(string(buf.buf), name, claimant, email);
     }
 
     /**
-     * @dev Submits a claim for an exact match (eg, fooeth.test -> foo.eth).
+     * @dev Submits a claim for prefix match (eg, fooeth.test -> foo.eth).
      *      Claimants must provide an amount of ether equal to 365 days'
      *      registration cost; call `getClaimCost` to determine this amount.
      *      Claimants should supply a little extra in case of variation in price;
      *      any excess will be returned to the sender.
      * @param name The DNS-encoded name of the domain being used to support the
      *             claim.
      * @param claimant The address of the claimant.
+     * @param email An email address for correspondence regarding the claim.
      */
-    function submitPrefixClaim(bytes memory name, address claimant) public payable {
+    function submitPrefixClaim(bytes memory name, address claimant, string memory email) public payable {
         bytes memory firstLabel = bytes(getLabel(name, 0));
         require(firstLabel.equals(firstLabel.length - 3, bytes("eth")));
-        handleClaim(string(firstLabel.substring(0, firstLabel.length - 3)), name, claimant);
+        handleClaim(string(firstLabel.substring(0, firstLabel.length - 3)), name, claimant, email);
+    }
+
+    /**
+     * @dev Closes the claim submission period.
+     *      Callable only by the owner.
+     */
+    function closeClaims() external onlyOwner inPhase(Phase.OPEN) {
+        phase = Phase.REVIEW;
+    }
+
+    /**
+     * @dev Ratifies the current set of claims.
+     *      Ratification freezes the claims and their resolutions, and permits
+     *      them to be acted on.
+     */
+    function ratifyClaims() external inPhase(Phase.REVIEW) {
+        // Only the ratifier can ratify.
+        require(msg.sender == ratifier);
```

Comment:
any reason owner cannot call this function (given that it's allowing owner to `setClaimStatus` at l194)?

Reply:
Because if the owner could ratify, there'd be no point in having ratification.